src/base/GlobalConfig.ml

```diff
@@ -144,6 +144,10 @@ let file_extn_library = "scillib"
 (* File extension for Scilla expressions. *)
 let file_extn_expression = "scilexp"
 
+(* Reset internal state. *)
+let reset () =
+  stdlib_dirs := []
+
```

Comment:
This should also include the other `ref`s in the file.

Reply:
To make things look consistent I suggest adding another `reset` function that will call the `StdlibTracker.reset ()` along with resetting other `refs`.  So the `StdlibTracker.reset ()` resets only things related to the `StdlibTracker` module and `GlobalConfig.reset ()` resets the whole `GlobalConfig`. What do you think?

Reply:
I think that `Datatypes.DataTypeDictionary.reinit ()` could be moved to the `GlobalConfig.reset ()` too